File: src/main/java/com/ocbc/auctionservice/controllers/HealthController.java
//package com.ocbc.auctionservice.controllers;
//
//import com.ocbc.auctionservice.services.HealthService;
//import org.springframework.beans.factory.annotation.Autowired;
//import org.springframework.http.ResponseEntity;
//import org.springframework.stereotype.Controller;
//import org.springframework.web.bind.annotation.GetMapping;
//import org.springframework.web.bind.annotation.RequestMapping;
//
//@Controller
//@RequestMapping("api/v1/health")
//public class HealthController {
//
//    @Autowired
//    private HealthService healthService;
//
//    @GetMapping
//    public ResponseEntity<String> health(){
//        return ResponseEntity.ok(healthService.healthCheck());
//    }
//}

File: src/main/kotlin/com/auction/service/controllers/HealthController.kt
package com.auction.service.controllers

import org.springframework.http.ResponseEntity
import org.springframework.http.ResponseEntity.*
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/api/v1/health")
class HealthController{

    @GetMapping
    fun health(): ResponseEntity<String> = ok("I am here!")
}